### ISSUE TITLE
Accessibility: Minicart, shipping select has no accessible name

### ISSUE THREAD

**Issue by msiewierska** [shopware/platform#13977]
### Shopware Version

6.7.3.1

### Affected area / extension

Platform(Default)

### Actual behaviour

WCAG 2.1 success criterion A is violated: 4.1.2 (level A), shipping method select has no accessible name

<img width="512" height="129" alt="Image" src="https://github.com/user-attachments/assets/6099bf9d-18e0-47b8-a99a-7fca630a8bfa" />

### Expected behaviour

Shipping select has accessible name.

### How to reproduce

Add product to cart, open minicart, inspect shipping select
